.env

```diff
@@ -5,3 +5,7 @@ REDIS_URL=redis://localhost:6379/0
 REDIS_PROVIDER=REDIS_URL
 # For Rails.cache if it's wanted/needed
 REDIS_CACHE_URL=redis://localhost:6379/1
+# For puma
+PORT=3000
+# For connecting to Postgres
+DATABASE_URL=postgres://fpc:fpc@localhost/fpc_develop
```

Comment:
This line actually broke my local setup (as my database is running somewhere else). I think it would be better to move this to `.env.local` and mention it in the setup instructions.